lib/rucio/core/account.py

```diff
@@ -236,9 +288,13 @@ def has_account_attribute(account, key, *, session: "Session"):
 
     :returns: True or False
     """
-    if session.query(models.AccountAttrAssociation.value).filter_by(account=account, key=key).first():
-        return True
-    return False
+    query = select(
+        models.AccountAttrAssociation
```

Comment:
This changes the behavior in terms that it will return `True` on an attribute with value `None`. I don't know if this problematic or if that was an undesired behavior before.

Reply:
I don’t remember exactly what my thinking was. I might have not realised the change in behaviour. The documentation string suggests the `has_account_attribute()` should only care about the existence of the attribute and not its value, but I will make it as before because this PR is supposed to maintain exact behaviour. Thanks for bringing this up.